### DISCUSSION TITLE
Is it possibile to Sync to a Nextcloud instance?

### DISCUSSION THREAD

**Discussion by kintaro1981** [immich-app/immich#2784] [Q&A]
Hello, I have a hetzner storage share instance (Nextcloud) and I would like to know if it’s possibile with immich to sync to nextcloud like photosync does.

I know that immich is not the same as photosync, but maybe some features overlaps?

**Comment by alextran1502** [comment]
You can point UPLOAD_LOCATION to your Nextcloud instance and Immich will upload the files to that location

**Reply by muava12** [reply]
can upload location detect photo inside subfolder?

**Reply by metzger100** [reply]
The Problem is, that nextcloud won't notice that there are new Images in the nextcloud folder. You have to create a cron job or something like that which checks for changes in a regular basis to add it to the nextcloud database

**Reply by kintaro1981** [reply]
Why NC is not able to notice changes?

**Reply by metzger100** [reply]
Because it only updates the database when triggered. It doesn't scan for changes of the files by default. You have to trigger it.

**Reply by Yetangitu** [reply]
That is easily solved by running nextcloud-inotifyscan:

https://github.com/Blaok/nextcloud-inotifyscan

**Reply by kintaro1981** [reply]
Thank you @Yetangitu, sadly is a solution only for who self host nextcloud.
I doubt Hetzner is going to add nextcloud-inotifyscan to their storage share. :-(
I'm going to try to suggest it to Hetzner.

**Comment by boterocamilo** [comment]
@kintaro1981 did you find a way for doing it?
I'm looking for the same feature. ✌️

**Reply by kintaro1981** [reply]
I didn't try. But @alextran1502 is saying is possibile.
@muava12 have you tried if it detect subfolders?

**Comment by klendgit** [comment]
It should also be possible to mount nextcloud pictures as a webdav folder on your server and have it available in immich via the EXTERNAL_PATH construction. I use that method with a Hetzner Storage Box (no nextcloud) and sshfs

**Reply by Sulmuza** [reply]
unfortunately webdav is too slow

**Reply by kintaro1981** [reply]
Slow but fully working subdirectories included?

**Comment by dvux1988** [comment]
I wanna try the same thing - Hetzner NC and immich as "app". Ist there a way now?
